server/base/src/main/java/org/apache/accumulo/server/init/Initialize.java

```diff
@@ -629,6 +629,8 @@ private static void initZooKeeper(Opts opts, String uuid, String instanceNamePat
     zoo.putPersistentData(zkInstanceRoot, EMPTY_BYTE_ARRAY, NodeExistsPolicy.FAIL);
     zoo.putPersistentData(zkInstanceRoot + Constants.ZTABLES, Constants.ZTABLES_INITIAL_ID,
         NodeExistsPolicy.FAIL);
+    zoo.putPersistentData(zkInstanceRoot + Constants.ZTABLE_CONFIG_VERSION, new byte[0],
```

Comment:
Upgrade code will also need to create node.

Reply:
I am working on the upgrade code part now.

Reply:
> Upgrade code will also need to create node.

Reply:
Just the  Upgrader9to10.upgradeZookeeper class.